tests/cupy_tests/math_tests/test_misc.py

```diff
@@ -14,7 +14,7 @@ class TestMisc(unittest.TestCase):
     @testing.numpy_cupy_allclose(atol=1e-5)
     def check_unary(self, name, xp, dtype, no_bool=False):
         if no_bool and numpy.dtype(dtype).char == '?':
-            return 0
+            return numpy.int_(0)
```

Comment:
`testing.numpy_cupy_allclose` needs numpy array/scalar instance for cheking dtype.